vnext/Shared/OInstance.cpp

```diff
@@ -198,6 +198,18 @@ void InstanceImpl::SetInError() noexcept {
   m_isInError = true;
 }
 
+bool shouldStartHermesInspector(DevSettings &devSettings) {
+  bool isHermes =
+      ((devSettings.jsiEngineOverride == JSIEngineOverride::Hermes) ||
+       (devSettings.jsiEngineOverride == JSIEngineOverride::Default && devSettings.jsiRuntimeHolder &&
```

Comment:
wouldn't this be enough? i.e. if jsiEngineOverride is Hermes, then the jsiRuntimeHolder will say Hermes, right?

Reply:
Alex.. There is some redundency between these two fields in devsettings .. Users can set devsettings.jsiEngineOverride to allow us to create the jsi runtime internally ... or provide a jsi runtime explicitly using devsettings.jsRuntimeHolder .. In the latter case, devsettings.jsiEngineOverride stays as Default as of now..

We could as consumers to set (or somehow set automatically) devsettings.jsiEngineOverride  too when using devsettings.jsRuntimeHolder, but we need some static asserts to ensure they do that before assuming it ..

Reply:
@asklar I hope you agreed ! :) Could you sign off on this change ?

Reply:
got it - so this is to support the private devsettings api to provide a runtime holder. If so, do you need to check that the override is default? why not just check for the runtimeholder && getRuntimeType?